### PR TITLE
upcoming: [DI-22596] - Overview and Criteria changes for ACLP Alerts

### PR DESCRIPTION
**Pull request by venkymano-akamai**

### CODE REVIEW

File: packages/manager/src/features/CloudPulse/Alerts/constants.ts
Comment (santoshp210-akamai) [inline packages/manager/src/features/CloudPulse/Alerts/constants.ts:1]: @venkymano-akamai , We already have the Item<string, {value_type}>[] lists defined for the mappings of severity, operators, aggregation types. We can use the same ones for reverse-mapping. Seems redundant to have two sets of maps for same data.

Comment (santoshp210-akamai) [reply packages/manager/src/features/CloudPulse/Alerts/constants.ts:1]: For the dimension filter operator values, the list is in the current PR that's active. That will also be pushed.

File: packages/manager/src/features/CloudPulse/Alerts/Utils/utils.ts
Comment (santoshp210-akamai) [inline packages/manager/src/features/CloudPulse/Alerts/Utils/utils.ts]: This should be fine, but we can use the tokens.color notation for this
```suggestion
    theme.name === 'light' ? theme.tokens.color.Neutrals[5] : theme.tokens.color.Neutrals.Black
```

File: packages/manager/src/features/CloudPulse/Alerts/AlertsDetail/AlertDetailChips.tsx
Comment (nikhagra-akamai) [inline packages/manager/src/features/CloudPulse/Alerts/AlertsDetail/AlertDetailChips.tsx:23]: can we rename them to more clear name because they are not actually width but grid columns to occupy or may more clear description to each of them

**Review by nikhagra-akamai — APPROVED** [review]

**Review by santoshp210-akamai — APPROVED** [review]

File: packages/manager/src/features/CloudPulse/Alerts/constants.ts
Comment (venkymano-akamai) [reply packages/manager/src/features/CloudPulse/Alerts/constants.ts:1]: @santoshp210-akamai , lets merge both and take a final call after that

File: packages/manager/src/features/CloudPulse/Alerts/Utils/utils.ts
Comment (venkymano-akamai) [reply packages/manager/src/features/CloudPulse/Alerts/Utils/utils.ts]: updating

**Comment by venkymano-akamai** [comment]
@santoshp210-akamai / @nikhagra-akamai - merging this since i have 2 approvals

**Pull request merged**
